doc/atcommands.txt

```diff
@@ -1808,20 +1808,19 @@ Destroys the specified channel.
 
 ---------------------------------------
 
-@fontcolor <color_name>
+@fontcolor <color_hex>
```

Comment:
By using clif colors we could make it color name or hex to not break support for existing scripts/commands.